pkg/clusteroperator/clusteroperator.go

```diff
@@ -25,11 +28,11 @@ const (
 // Builder provides struct for clusterOperator object.
 type Builder struct {
 	// ClusterOperator definition. Used to create a clusterOperator object.
-	Definition *v1.ClusterOperator
+	Definition *configV1.ClusterOperator
```

Comment:
Please refactor it to *configv1.ClusterOperator

Reply:
but for the clint.go it is configV1, why?

Reply:
We'll refactor clients.go other packages later. Now we are trying to switch to small letters in import